packages/website/plugins/generated-rule-docs/insertions/insertNewRuleReferences.ts

```diff
@@ -54,18 +54,65 @@ export async function insertNewRuleReferences(
     child => nodeIsHeading(child) && child.depth === 2,
   );
 
-  const eslintrc = `{
-  "rules": {
+  const rules = `{
     "@typescript-eslint/${page.file.stem}": "error"
-  }
+  }`;
+
+  const eslintrc = `{
+  "rules": ${rules}
+}`;
+
+  const eslitnConfig = `{
```

Comment:
```suggestion
  const eslintConfig = `{
```

Reply:
I fixed it in [c2aad1c](https://github.com/typescript-eslint/typescript-eslint/pull/10164/commits/c2aad1c8b0c321cb51ae4a31dd0d91b96a0a8beb).